components/bl-transitions-component/src/others.js

```diff
@@ -54,23 +56,30 @@ export const Others = props => {
 
   useEffect(() => {
     if (element.current) {
-      element.current.classList.add('transition', variants);
+      element.current.classList.add('transition', variant);
 
       if (display && isTransition) {
-        element.current.classList.add(variants + '--active');
-        element.current.style.transitionDuration = duration + 'ms';
-
+        onOpen(element, variant, duration);
         showElement(rootRef.current);
       } else {
-        element.current.classList.remove(variants + '--active');
-        element.current.style.transitionDuration = 0;
+        onHide(element, variant);
       }
     }
-  }, [element, variants, display, isTransition, duration]);
+  }, [element, variant, display, isTransition, duration]);
 
   return (
-    <div ref={ rootRef } className={ cn('bl-customComponent-transitions', variants, classList) } style={ style }>
+    <div ref={ rootRef } className={ cn('bl-customComponent-transitions', variant, classList) } style={ style }>
       { transitionsContainerPod.render() }
     </div>
   );
 };
+
+const onOpen = (element, variant, duration) => {
```

Comment:
Here shouldn't be current proiperty, function should work with clear html element.